Testing/Functional/Products/TestPlans/aad.g5.testplan.yaml

```diff
@@ -1,7 +1,7 @@
 ProductName: aad
 TestPlan:
   - PolicyId: MS.AAD.2.1v1
-    TestDriver: RunCached
+    TestDriver: ScubaCached
```

Comment:
Please take a look here: https://github.com/cisagov/ScubaGear/blob/ab4036e4801cf9e03c4d5c786bf70264e2701a27/Testing/Functional/Products/Products.Tests.ps1#L252

I think you will need to match the case.

also, please add a comment to that file as James suggested clarifying that those strings need to match "TestDriver" in the test plans.